README.md

```diff
@@ -1,3 +1,280 @@
 # sf-functions-python
 
-Python support for Salesforce Functions.
+Python support for [Salesforce Functions](https://developer.salesforce.com/docs/platform/functions/overview).
+
+> Note: This feature is in beta and has been released early so we can collect feedback. It may contain significant problems, undergo major changes, or be discontinued. The use of this feature is governed by the Salesforce.com Program Agreement.
+
+---
+
+# Getting Started with Python for Functions
+
+## Prerequisites
+
+> Any install commands that follow assume a Mac OSX system with Homebrew available.  If you’re on another OS you’ll have to click through the links to get alternative install instructions.
+
+### [Install Python 3](https://www.python.org/downloads/)
```

Comment:
This heading links to the Python site, which only has downloads for Windows.

Should we move this link out of the heading and put it alongside the Homebrew link below (and also explicitly label it as for other OSes), so they are side-by side?

Reply:
the page it leads me to provides an installer for my OS &  provides links out to other OS downloads.